MaterialLib/SolidModels/MFront/MFront.cpp

```diff
@@ -11,6 +11,8 @@
 
 #include <MGIS/Behaviour/Integrate.hxx>
 
+#include "NumLib/Exceptions.h"
```

Comment:
It might be good to move `Exceptions.h` to `BaseLib` or to ProcessLib/Utils by the name of `AssemblyException`.

Reply:
`BaseLib` would be too much exposure, because these (that) exception(s) are used only on `NumLib` level.
`ProcessLib/Utils` would not work, because of the include in `MaterialLib`, causing a circular dependency.

Reply:
I see. Then just keep it as it is.